weblate/templates/screenshots/screenshot_show.html

```diff
@@ -1 +1,2 @@
-<a href="{{ screenshot.image.url }}" class="thumbnail" alt="{{ screenshot.name }}" title="{{ screenshot.name }}" data-edit="{% url 'screenshot' pk=screenshot.pk %}"><img class="img-responsive" src="{{ screenshot.image.url }}" /></a>
+{% perm 'screenshot.edit' screenshot.translation.component as user_can_edit_screenshot %}
+<a href="{{ screenshot.image.url }}" class="thumbnail" alt="{{ screenshot.name }}" title="{{ screenshot.name }}" data-details-url="{% url 'screenshot' pk=screenshot.pk %}"{% if user_can_edit_screenshot %} data-can-edit{% endif %}><img class="img-responsive" src="{{ screenshot.image.url }}" /></a>
```

Comment:
The permission is already checked in the parent template. While at it, we can also fix the `alt` attribute placement:

```suggestion
<a href="{{ screenshot.image.url }}" class="thumbnail" title="{{ screenshot.name }}" data-details-url="{% url 'screenshot' pk=screenshot.pk %}"{% if user_can_edit_screenshot %} data-can-edit{% endif %}><img class="img-responsive" src="{{ screenshot.image.url }}" alt="{{ screenshot.name }}" /></a>
```